utils/payment-actions.js

```diff
@@ -147,3 +147,9 @@ export function completePurchase() {
   )
   cy.waitAndGet('.payment-submit-wrap > button.submit:visible', 3000).click()
 }
+
+export function insertCoupon() {
+  cy.get('#cart-link-coupon-add').click({ force: true })
+  cy.get('#cart-coupon').type('carlos', { force: true })
```

Comment:
I'd prefer a string like "fakeCoupon" to be clearer for future maintainers.

Reply:
I agree! I just used this one because it had been created already. I'll create a new one. What do you think about it being `freeshipping`?

Reply:
Sounds good too.